### PR TITLE
Add exception to unsubscribe

### PR DESCRIPTION
**Pull request by mauricioirace**
Changes:
- Throw an exception when trying to unsubscribe from a topic you are not subscribed

### CODE REVIEW

File: src/api/CommunicationsApiImpl.ts [hunk 1]
```diff
@@ -35,18 +35,16 @@ class CommunicationsApiImpl implements CommunicationsApi {
...
             }
             callback();
         } catch (error) {
-            callback({subscribeError: {
-                channel: {
-                    channelId: ""
-                },
-                reason: error.message
-            }});
+            callback({code: grpc.status.NOT_FOUND, message: `Peer was not subscribed to ${subscription.topic.address}`});
```
Comment (santicomp2014) [inline src/api/CommunicationsApiImpl.ts]: This should be a unSubscribeError, update proto
thanks

**Review by santicomp2014 — DISMISSED** [review]
Left a comment to update proto

File: src/api/CommunicationsApiImpl.ts [hunk 2]
```diff
@@ -39,20 +39,23 @@ class CommunicationsApiImpl implements CommunicationsApi {
...
-                }
-            }, null);
+            const err = {subscribeError: {
+                channel: {
+                    channelId: ""
+                },
+                reason: error.message
+            }
+            }
+            callback({code: grpc.status.NOT_FOUND, message: JSON.stringify(err)});
```
Comment (mauricioirace) [inline src/api/CommunicationsApiImpl.ts]: I don't think lumino team will like a JSON.stringify

File: src/api/CommunicationsApiImpl.ts [hunk 3]
```diff
@@ -168,13 +171,13 @@ class CommunicationsApiImpl implements CommunicationsApi {
...
-            callback({
+        } catch (error) {
+           const err = {
                 connectCommsError: {
-                    reason: err.message
+                    reason: error.message
                 }
-            }, null);
+            }
+            callback({code: grpc.status.NOT_FOUND, message: JSON.stringify(err)});
```
Comment (mauricioirace) [inline src/api/CommunicationsApiImpl.ts]: the same as above

**Review by mauricioirace — COMMENTED** [review]
I don't think lumino team will like the stringify

**Pull request merged**